Sprint 1

TzviZeller opened the pull request:
- bin/bash --> usr/bin/bash (https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash)

- clear apt cache and upgrade

- install chrome

Commit:
- bin/bash --> usr/bin/bash (https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash)

- clear apt cache and upgrade

- install chrome

malexanderboyd commented on bootstrap.sh:
```diff
@@ -1,54 +1,67 @@
-#!/bin/bash
+#!/usr/bin/bash
```
Use `#!/usr/bin/env bash` for more portability, and searches users `PATH` for bash

Commit:
Init test with travis ci

Commit:
Fix .travis file

Commit:
Travis CI fix

Commit:
fix Ci instance v2

TzviZeller commented:
seems to be an issue with travis exiting so this is now my P0

malexanderboyd commented on bootstrap.sh:
```diff
@@ -57,8 +70,11 @@ cp ./rupza.vim ~/.vim/colors/rupza.vim
 #Reload Bash
 exec bash
+
+echo "All Done :)"
```
this is 2019, use emojis

TzviZeller replied:
but I am a hipster so no

malexanderboyd commented on bootstrap.sh:
```diff
@@ -1,54 +1,67 @@
 wget -qO - https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
 sudo sh -c 'echo "deb [arch=amd64] https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
 
+#Add Deb Server - Chrome
```
firefox pls

TzviZeller replied:
in your dreams

malexanderboyd commented on bootstrap.sh:
```diff
@@ -1,54 +1,67 @@
+#!/usr/bin/bash
 
-set -eux pipefail
+set -e  # Exit with nonzero exit code if anything fails
```
Did this end up working w/ -x?

TzviZeller replied:
Currently testing in newest push

TzviZeller replied:
@malexanderboyd still nothing, can you see the travis logs? If not will link it

malexanderboyd replied:
Yes, I can see now - Left comments around this

malexanderboyd replied:
@TzviZeller

malexanderboyd commented on bootstrap.sh:
```diff
@@ -1,54 +1,67 @@
+set -e  # Exit with nonzero exit code if anything fails
 
 echo "Validate sudo credentials ..."
 sudo -v
```
I think you could just attempt to install sudo here, worst case it doesn't install it since its already ready

TzviZeller replied:
Currently testing in newest push

TzviZeller replied:
?

malexanderboyd replied:
my mistake, I thought that `sudo -v` was to validate that sudo was installed, not to input sudo credentials.

malexanderboyd commented on bootstrap.sh:
```diff
@@ -1,54 +1,67 @@
+apm install todo-show
+apm install open-in-browser
+apm install highlight-selected
+#Configure Atom - Theme Install
```
Look at this [guide](https://code.visualstudio.com/docs/setup/linux) on how to do this! So dope!

TzviZeller replied:
atom for now but will try VS code

Commit:
v3 fix travis ci

malexanderboyd commented on bootstrap.sh:
```diff
@@ -1,54 +1,67 @@
+#!/usr/bin/bash
 
-set -eux pipefail
+set -x  # Exit with nonzero exit code if anything fails
```
This [example](https://docs.travis-ci.com/user/job-lifecycle#complex-build-commands) uses `set -ev`, maybe try that for failing CI ?

malexanderboyd commented on .travis.yml:
```diff
@@ -0,0 +1,6 @@
+language: bash
+
+script:
+    - bash bootstrap.sh
```
I don't think you need to use `bash ...` here, following [this](https://docs.travis-ci.com/user/job-lifecycle#complex-build-commands) they point to the script to run

Commit:
fix 4?

Commit:
fix spotify

Commit:
fix to a fix

Commit:
set exit code

Commit:
exit 0 --> travis_terminate 0

Commit:
ahahhhhh

Commit:
plz work

Commit:
no exec

Commit:
- finaly fixed error with Travic Ci, removed exec

- safer exits 
(https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/)

- standardizerd deb server addition

TzviZeller commented:
Closing as now we have testing built out
